src/puget/printer.clj

```diff
@@ -170,6 +171,17 @@
 
 ;; ## Formatting Methods
 
+(defn- pos-int?
+  [x]
+  (if-let [core-pos-int? (resolve 'clojure.core/pos-int?)]
```

Comment:
Is it possible to resolve this at compile time instead of at runtime? Alternately, it doesn't seem like there's much advantage to using the clojure core version at all given that the library still needs the implementation below.